### DISCUSSION TITLE
Installing a bundle from a previous version

### DISCUSSION THREAD

**Discussion by margotbignon** [symfony/symfony#53611] [Q&A]
Hi,

I'd like to install this bundle: https://github.com/agence-adeliom/easy-media-bundle
in my Symfony 7 project, but it's compatible with Symfony 6. Is there a way to modify the code so that I can use it in my project? 

Thanks for your help

**Comment by ghost** [comment]
I can see this bundle is in active development so you can open an issue there and ask for SF 7 compability release.
If you are in a hurry then you can fork this repository and update the version constraints of symfony/* dependencies and add ` || ^7.0` **HOWEVER** it's not guaranteed than bundle will work. It will if the bundle does not use code that was removed in 7.0, it will not otherwise.
